### ISSUE TITLE
Update to GUI

### ISSUE THREAD

**Issue by Sepero** [glutanimate/tempthrottlergui#1]
Hello Friend. I would like to know if you would be interested in updating/rewriting your code here? I would like to work with you on an update. The yad library is now included in the standard Ubuntu repositories, which would make it more easily available to all users. Perhaps reconstruct the GUI more modularized. Hope to hear back from you soon. Cheers.

**Comment by glutanimate** [comment]
Hey Sepero,

I'd love to help, but I've got my plate full with other projects right now. Sadly I don't see this changing for the foreseeable future. If I do manage to find some spare time I'll make sure to get back to you.

Thanks for the continuous work on temp-throttle, by the way. I really appreciate it!

**Comment by Sepero** [comment]
Yeah, I'm happy to keep temp-throttle updated. I love the project. As soon as I can, I'll be fixing a bug that affects arm.

About the GUI, it's no big hurry. I'm actually very busy myself now too haha. I intend that maybe we can collaborate on something during winter, or the beginning of next year. Contact me anytime. :)
